py/torch_tensorrt/dynamo/conversion/plugins/_generate_plugin.py

```diff
@@ -12,211 +11,222 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+try:
+    import tensorrt.plugin as trtp
 
-def mksym(
-    shape_env: ShapeEnv, value: int, source: LocalSource, dynamic_dim: DimDynamic
-) -> torch.SymInt:
-    return shape_env.create_symintnode(
-        shape_env.create_symbol(
-            value,
-            source=source,
-            dynamic_dim=dynamic_dim,
-        ),
-        hint=value,
-        source=source,
+    assert trtp
+except ImportError as e:
+    _LOGGER.warning(
+        "Unable to import TensorRT plugin. Please install TensorRT plugin library (https://github.com/NVIDIA/TensorRT-plugin-library?tab=readme-ov-file#installation) to add support for compiling quantized models"
     )
+else:
+
```

Comment:
Instead of a giant if else, take a look at https://github.com/pytorch/TensorRT/blob/c3ad86cdd51a4f9ac3b6cef383629c5c372862bf/py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py#L56, https://github.com/pytorch/TensorRT/blob/c3ad86cdd51a4f9ac3b6cef383629c5c372862bf/py/torch_tensorrt/_features.py#L53